CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ eth_policy()
 # project name and version should be set after cmake_policy CMP0048
 set(PROJECT_VERSION "0.6.12")
 # OSX target needed in order to support std::visit
-set(CMAKE_OSX_DEPLOYMENT_TARGET "10.14")
+set(CMAKE_OSX_DEPLOYMENT_TARGET "10.15")
```

Comment:
I just built fine with this at 10.14, you sure we need this?

Reply:
I've had issues with this when jumping between 0.5 and 0.6 branches; I'm gonna leave unless you think it breaks when you try it